src/core/proxy.js

```diff
@@ -19,7 +19,12 @@ const requestHandler = async (request, proxy, overrides = {}) => {
         agent: setAgent(proxy),
         responseType: "buffer",
         maxRedirects: 15,
-        throwHttpErrors: false
+        throwHttpErrors: false,
+        ignoreInvalidCookies:
+            typeof overrides.ignoreInvalidCookies === "undefined"
+                ? false
+                : overrides.ignoreInvalidCookies,
```

Comment:
Remove the conditional and just use `ignoreInvalidCookies: true,`

Reply:
Sounds good 👍